webapp-django/crashstats/crashstats/static/crashstats/css/base/layout.less

```diff
@@ -20,9 +20,14 @@ html {
             align-items: center;
 
             h2 {
-                flex-grow: 1;
+                flex: auto;
```

Comment:
This fixes the flex for report view where the signature can make the line go reaaaaaaaaaaly long. Now it doesn't stomp on the navigation.